src/App.js

```diff
@@ -1,5 +1,65 @@
+import { Console } from "@woowacourse/mission-utils";
+
 class App {
-  async run() {}
+  async run() {
+    try {
+      let input = await Console.readLineAsync("덧셈할 문자열을 입력해주세요.");
+
+      if (input.trim() === "") {
+        Console.print(0);
+        return;
+      }
+      const isQuote = (input) => /['"]+/.test(input);
+      const isPositive = (input) => /^[+]?[1-9]\d*(\.\d+)?$/.test(input);
+
+      let delimiters = [",", ":"];
+
```

Comment:
```suggestion
      let delimiters = [",", ":"];
```
이 부분은 const로 지정해도 괜찮습니다.

Reply:
> 재할당하지 않고 push로 배열의 원소를 추가했기 때문에 const로 지정해도 괜찮습니다.

아 원소를 push로 추가하면 const로 선언해도되는군요 하나 배웁니다 🙏

Reply:
push는 const를 사용해도 되는군요 감사합니다!